src/client/automation/playback/click/index.js

```diff
@@ -178,8 +178,7 @@ export default class ClickAutomation extends VisibleElementAutomation {
     }
 
     _mouseup (eventArgs) {
-        return cursor
-            .buttonUp()
+        return cursor.buttonUp()
```

Comment:
Save origin formatting here and below.
```js
cursor
    .buttonUp()
    .then
```

Reply:
Why do we need break the function call expression?

```js
a
    .add(b); // strange

// instead of
a.add(b);
```

Also, I can't see the function call context when I search 'getSomething',  in the search panel. I see one line, and can't see getSomething of what?